mne/preprocessing/maxwell.py

```diff
@@ -503,6 +518,36 @@ def maxwell_filter(raw, origin='auto', int_order=8, ext_order=3,
     return raw_sss
 
 
+def _get_coil_scale(meg_picks, mag_picks, grad_picks, mag_scale, info):
+    """Helper to get the magnetometer scale factor"""
+    if isinstance(mag_scale, string_types):
+        if mag_scale != 'auto':
+            raise ValueError('mag_scale must be a float or "auto", got "%s"'
+                             % mag_scale)
+        if len(mag_picks) in (0, len(meg_picks)):
+            mag_scale = 100.  # only one coil type, doesn't matter
+            logger.info('    Setting mag_scale=%0.2f because only one '
+                        'coil type is present' % mag_scale)
+        else:
+            # Find our gradiometer baseline
+            coils = _create_meg_coils(pick_info(info, meg_picks)['chs'],
+                                      'accurate')
+            grad_base = set(coils[pick]['base'] for pick in grad_picks)
```

Comment:
Do you mind modifying the `# Find our gradiometer baseline` line to indicate that it's related to the physical sensor size? Also, Matti refers to this as the "base line". Maybe something like:
`# Find our gradiometer "base line" (physical distance between pickup loops)`

Other than that, LGTM